src/components/popover-container/popover-container-test.stories.tsx

```diff
@@ -97,3 +104,131 @@ export const PopoverContainerWithSelect = () => {
     </div>
   );
 };
+
+export const InAScrollableBlock = () => {
+  const [value1, setValue1] = useState(false);
+  const [value2, setValue2] = useState(false);
+  const [value3, setValue3] = useState(false);
+  const [value4, setValue4] = useState(false);
+  const Logo = () => <img height={28} src={carbonLogo} alt="Carbon logo" />;
+  return (
+    <Box>
```

Comment:
**comment** (non-blocking): I'm probably missing something, but as far as I understand it we only need the PopoverContainer here and the scrollable Box that wraps it. I don't see what value the GlobalHeader, menus and TileSelect add to the story - indeed the TileSelect confused me when i was first looking at it (especially as I was on the Docs tab and there you don't even seem to be able to see or scroll to the PopoverContainer that's below it! It's fine on Canvas though, so not an issue as this is only for internal QA purposes.)

Reply:
The original issue was composed using the `GlobalHeader`. The original issue had a GIF and screenshot of a live product so I removed it. I'll send you the examples privately and that might give you more context.